DAO.sol

```diff
@@ -84,7 +84,7 @@ contract DAOInterface {
     uint public proposalDeposit;
 
     // the accumulated sum of all current proposal deposits
-    uint sumOfProposalDeposits;
+    uint public sumOfProposalDeposits;
```

Comment:
Then perhaps make the `actualBalance()` as external. Since it would give the true balance of the DAO, while with `sumOfProposalDeposits` being `public` the user would need to also do a mental subraction to get that amount.